groups/forms.py

```diff
@@ -45,17 +46,21 @@ class DiscussionSubscribeForm(forms.Form):
 
     helper = FormHelper()
     helper.form_class = 'form-horizontal'
-    helper.action = reverse_lazy('discussion-subscribe')
+    helper.form_tag = True
 
     class Meta:
         fields = ('subscribe',)
 
     def __init__(self, *args, **kwargs):
         """
-        Accept a value for the 'subscribe' variable, and build the layout
-        to reflect the action the form will take.
+        Build the layout to reflect the action the form will take.
+
+        Accepts (and requires) a user and a discussion as keyword arguments.
         """
-        to_subscribe = kwargs.pop('subscribe', True)
+        user = kwargs.pop('user')
+        discussion = kwargs.pop('discussion')
```

Comment:
@Ian-Foote Yeah, I know.  I thought @meshy was specifically asking for the keyword-only version.